### PR TITLE
[kube-prometheus-stack]: Document cert-manager usage

### PR DESCRIPTION
**Pull request by MythicManiac**
Add instructions on how to enable admission webhook TLS certificate provisioning via cert-manager to the readme.

### CODE REVIEW

**Comment by MythicManiac** [comment]
I don't think the CI failure regarding chart bump is valid here given that it's a README change?

**Comment by MythicManiac** [comment]
Let me know if you feel like signoff and/or chart version bump is actually required in this case. Since it's just a documentation change I personally feel like those might be a bit overkill, but I'll gladly follow suit if that's how this project rolls

**Review by monotek — CHANGES_REQUESTED** [review]
Please raise chart version and sign your commits.

**Comment by MythicManiac** [comment]
@monotek done

**Review by monotek — APPROVED** [review]

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 12.12.1
+version: 12.12.2
```
Comment (gkarthiks) [inline charts/kube-prometheus-stack/Chart.yaml:21]: @MythicManiac can you please bump the chart version

**Review by bismarck — APPROVED** [review]
lgtm

**Pull request merged**